llama-index-core/llama_index/core/ingestion/pipeline.py

```diff
@@ -732,9 +735,6 @@ async def arun(
                 **kwargs,
             )
 
-        if self.vector_store is not None:
-            await self.vector_store.async_add(
-                [n for n in nodes if n.embedding is not None]
-            )
+        self._handle_add_to_vector_store(self, nodes)
```

Comment:
now this doesn't call `async_add()` -- probably don't need the helper method for this change

Reply:
fixed